docs/src/visualization/sharing.md

```diff
@@ -120,5 +110,5 @@ With Nextstrain Groups, you can choose whether each dataset is publicly viewable
 #### Privacy and security  
 Independently hosted Auspice servers can be configured with any security protocols necessary.
 
-## [Previous Section: Orientation: Customizing your visualization](customizing-visualization.md)
+## [Previous Section: Orientation: Customizing your visualization](../analysis/customizing-visualization.md)
```

Comment:
Now that we are in RTD where the documentation system knows what the previous and next pages are, we should consider dropping these H2 header links.

Reply:
Yes - let's remove these